maintaining-and-tuning/monitoring.md

```diff
@@ -162,15 +162,45 @@ threadpool_threads{job="grr_worker"}
 ```	
 rate(grr_client_crashes_total{job="grr_worker"}[5m])
 ```
-### Example visualization and alerting setup
+
+## Real-World Setup
```

Comment:
"Real-World" is a confusing term here. Does the rest of the doc describe something other than the real world setup? :) I understand that the intention here is to talk about security and implications of using Prometheus/Grafana in larger setups. But then I'd structure the whole document as:
1. Here's how to setup Prometheus.
1. Prometheus security (and maybe scaling).
1. Here's how to setup Grafana.
1. Grafana security.